rom/dev/src/fuse.rs

```diff
@@ -0,0 +1,68 @@
+/*++
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    fuse.rs
+
+Abstract:
+
+    The file contains Fuse-related Implementations.
+
+--*/
+use caliptra_common::{FuseLogEntry, FuseLogEntryId};
+use caliptra_drivers::CaliptraResult;
+use caliptra_error::caliptra_err_def;
+use zerocopy::AsBytes;
+
+caliptra_err_def! {
+    RomGlobal,
+    GlobalErr
+    {
+        FuseLogInvalidEntryId = 0x6,
+        FuseLogUnsupportedDataLength = 0x7,
+    }
+}
+
+extern "C" {
+    static mut FUSE_LOG_ORG: u8;
+}
+
+/// Log Fuse data
+///
+/// # Arguments
+/// * `entry_id` - log entry ID
+/// * `data` -  data To log to the fuse log
+///
+/// # Return Value
+/// * `Ok(())` - Success
+/// * `Err(GlobalErr::FuseLogInvalidEntryId)` - Invalid Fuse log entry ID
+/// * `Err(GlobalErr::FuseLogUpsupportedDataLength)` - Unsupported data length
+///
+pub fn log_fuse_data(entry_id: FuseLogEntryId, data: &[u8]) -> CaliptraResult<()> {
```

Comment:
How should FMC/RT know the fuse log size?

Reply:
We have the entire fuse log as 1KB and this is always constant. Each entry is constant in size. If the FMC/RT needs to parse the log we can always use these constants to parse it. Don't think that this will change. Even if the sizes change, still the same methods can be applied to parse the logs. Pretty sure that we will never support variable size entries or the log size.

Reply:
See suggestion below for how FMC/RT could figure out how many entries there are